includes/class-wsuwp-shortcode-fundselector.php

```diff
@@ -69,13 +76,28 @@ public function fundselector_create_shortcode( $atts ) {
 		</div>';
 
 		// Unit Priorities Tab
-		$unit_priorities = $this->wsuf_fundselector_funds_get_funds( $args['unit_taxonomy'], $args['unit_category'] );
 		$unit_priorities_list = '<option disabled selected value> -- Select a Fund -- </option>';
 
-		foreach ( $unit_priorities as $unit_priority ) {
-			$fund_name = esc_html( $unit_priority['fund_name'] );
-			$fund_designation_id = esc_attr( $unit_priority['designation_id'] );
-			$unit_priorities_list .= "<option value=\"{$fund_designation_id}\">{$fund_name}</option>";
+		if ( rest_url( '' ) !== $args['rest_url'] ) { // If the REST URL was overridden
+			$plugin_api_url = esc_url( $args['rest_url'] . 'idonate_fundselector/v1/funds/' . $args['unit_taxonomy'] . '/' . $args['unit_category'] );
+			$response = wp_remote_get( $plugin_api_url );
```

Comment:
There is no technical reason stopping us from doing that. I thought that there may some performance improvements by doing it on the back-end, but that might not be true.

I will make the change to have it pull from JS and update the branch.